<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_claculateProj</class>
 <widget class="QWidget" name="dialog_claculateProj">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1284</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButtonAddTask">
         <property name="text">
          <string>Добавить задачу</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonAddSubTask">
         <property name="text">
          <string>Добавить подзадачу</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonRemoveTask">
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QTreeWidget" name="treeWidgetProject">
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string/>
        </property>
       </column>
       <item>
        <property name="text">
         <string>ПРОЕКТ</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="10" column="0" colspan="3">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="1" column="2">
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>- из неё на тестирование:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_25">
           <property name="text">
            <string>- из неё страховка:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>- из неё на разработку:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLineEdit" name="lineEditPriceForDevelopment">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="labelFPforTest">
           <property name="text">
            <string>0.00 р.</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="labelFPforEnsure">
           <property name="text">
            <string>0.00 р.</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="0" rowspan="3">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>Стоимость с наценкой:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" rowspan="3">
          <widget class="QLabel" name="labelFullPrice">
           <property name="text">
            <string>0.00 р.</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0" colspan="3">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="3">
          <widget class="QLineEdit" name="lineEditTimeForDevelopment">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>- из него на тестирование:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>- из него на разработку:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="labelTTKforTest">
           <property name="text">
            <string>0 дней 00 часов 00 минут</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="4">
          <widget class="Line" name="line">
           <property name="lineWidth">
            <number>2</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="1" rowspan="3">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>0 дней 00 часов 00 минут</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0" rowspan="3">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Время на реализацию:   </string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QLabel" name="labelTTKforEnsure">
           <property name="text">
            <string>0 дней 00 часов 00 минут</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>- из него на страховку:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="4">
          <widget class="Line" name="line_2">
           <property name="lineWidth">
            <number>2</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Себестоимость:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Наименование текущей задачи:</string>
         </property>
        </widget>
       </item>
       <item row="13" column="0" colspan="3">
        <widget class="QLabel" name="label_27">
         <property name="text">
          <string>Описание и коментарии к задаче:</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0" colspan="3">
        <widget class="QPlainTextEdit" name="plainTextEditDescription"/>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QListWidget" name="listWidget"/>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QLineEdit" name="lineEditCurrTask">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLabel" name="labelParentTaskName">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Список дочерних задач:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLabel" name="labelProjNameText">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="7" column="1" colspan="2">
        <widget class="QLabel" name="labelCostPrice">
         <property name="text">
          <string>0.00 р.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="8" column="1" colspan="2">
        <widget class="QLabel" name="labelMargin">
         <property name="text">
          <string>0.00 р.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Наценка:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Наименование проекта:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Наименование родительской задачи:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="3">
        <widget class="Line" name="line_3">
         <property name="lineWidth">
          <number>2</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="11" column="0" colspan="3">
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
